### ISSUE TITLE
FanControl does not start properly when CorsairLink is placed in plugins folder

### ISSUE THREAD

**Issue by hexcyan** [EvanMulawski/FanControl.CorsairLink#166]
When trying to run FanControl with CorsairLink in the plugins folder, it will show up in task manager but nothing will pop up in the foreground, or the tray. FanControl runs fine without the CorsairLink plugin.

I have unblocked all the files, and I have tried with the latest beta ([v1.6.0-beta.6](https://github.com/EvanMulawski/FanControl.CorsairLink/releases/tag/v1.6.0-beta.6)) and the latest release ([v1.5.1](https://github.com/EvanMulawski/FanControl.CorsairLink/releases/tag/v1.5.1)) of CorsairLink with the latest version ([V186](https://github.com/Rem0o/FanControl.Releases/releases/tag/V186)) of FanControl and and an older version ([V174](https://github.com/Rem0o/FanControl.Releases/releases/tag/V174)) with the same behaviours.

**Comment by EvanMulawski** [comment]
Hi @hexcyan, can you attach the `CorsairLink.log` and `log.txt` files (located in the Fan Control directory)? Thanks!

**Comment by hexcyan** [comment]
Hi, it doesn't seem like either of these files have been created

**Comment by hexcyan** [comment]
Figured out the issue, Citrix Workspace was causing issues as per this [support article](https://help.corsair.com/hc/en-us/articles/14242499752589-iCUE-Device-recognition-issue-with-Citrix-Workspace-installed). I suspected this was the case as it has caused other issues for me in the past and iCUE wasn't detecting my fan controllers either.

After uninstalling Citrix Workspace, both FanControl is now launching and loading CorsairLink correctly and iCUE is detecting the fan controllers again.
